lib/gc.js

```diff
@@ -202,12 +199,7 @@ GarbageCollector.prototype = {
     }
   },
 
-  scheduleSweep: function (interval) {
-    this.sweepTimeoutId = setTimeout(this.sweep.bind(this), interval);
-  },
-
-  sweep: async function () {
-    clearTimeout(this.sweepTimeoutId);
+  sweep: async function (full=false) {
```

Comment:
Since removing images only when there are no running tasks is a gc policy, I think makes more sense pass the number of running tasks to `sweep` and let the function decide how it wants to handle it.